pkg/internal/cryptoutil/utils.go

```diff
@@ -25,6 +25,24 @@ var errEmptyRecipients = errors.New("empty recipients")
 // errInvalidKeypair is used when a keypair is invalid
 var errInvalidKeypair = errors.New("invalid keypair")
 
+// SignatureAlgorithm represents a signature algorithm.
+type SignatureAlgorithm string
+
+// ContentEncryption represents a content encryption algorithm.
+type ContentEncryption string
```

Comment:
The naming compared to L29 seems inconsistent

Reply:
updated